metadata-ingestion/source_docs/snowflake.md

```diff
@@ -122,58 +132,63 @@ Note that a `.` is used to denote nested fields in the YAML recipe.
+| `include_table_lineage`        |          | `True`                                                                     | If enabled, populates the snowflake table-to-table and s3-to-snowflake table lineage. Requires appropriate grants given to the role.                                                                |
+| `include_view_lineage`         |          | `True`                                                                     | If enabled, populates the snowflake view->table and table->view lineages (no view->view lineage yet). Requires appropriate grants given to the role, and `include_table_lineage` to be `True`.     |
 | `bucket_duration`              |          | `"DAY"`                                                                    | Duration to bucket lineage data extraction by. Can be `"DAY"` or `"HOUR"`.                                                                                                              |
 | `start_time`                   |          | Start of last full day in UTC (or hour, depending on `bucket_duration`)    | Earliest time of lineage data to consider. For the bootstrap run, set it as far back in time as possible.                                                                               |
 | `end_time`                     |          | End of last full day in UTC (or hour, depending on `bucket_duration`)      | Latest time of lineage data to consider.                                                                                                                                                |
 | `profiling`                    |          | See the defaults for [profiling config](./sql_profiles.md#Config-details). | See [profiling config](./sql_profiles.md#Config-details).                                                                                                                               |
 | `domain.domain_key.allow`      |          |                                                                            | List of regex patterns for tables/schemas to set domain_key domain key (domain_key can be any string like `sales`. There can be multiple domain key specified.                          |
 | `domain.domain_key.deny`       |          |                                                                            | List of regex patterns for tables/schemas to not assign domain_key. There can be multiple domain key specified.                                                                         |
 | `domain.domain_key.ignoreCase` |          | `True`                                                                     | Whether to ignore case sensitivity during pattern matching.There can be multiple domain key specified.                                                                                  |
+| `setup.enabled`                |          | `False` | Whether setup of Snowflake role (used for ingestion) is enabled or not |
```

Comment:
reading the config, i didn't understand that this was about setting up a "role".

please make that more clear in the config names

Reply:
it should be provision_role or something

Reply:
changed